sdk/python/resources/workspace/workspace-managed-network.ipynb

```diff
@@ -0,0 +1,366 @@
+    "- `description` - Description of the workspace\n",
+    "- `tags` - (Optional) Tags to help search/filter on workspace easily\n",
+    "\n",
+    "Using the `MLClient` created earlier, we will create the workspace. This command will start workspace creation and provide a confirmation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name": "basic_workspace_name"
+   },
+   "outputs": [],
+   "source": [
+    "# Creating a unique workspace name with current datetime to avoid conflicts\n",
+    "\n",
+    "mvnet_workspace_name = \"mlw-mvnet-prod-\" + datetime.datetime.now().strftime(\n",
+    "    \"%Y%m%d%H%M\"\n",
+    ")\n",
+    "\n",
+    "ws_mvnet = Workspace(\n",
+    "    name=mvnet_workspace_name,\n",
+    "    location=\"eastus\",\n",
+    "    display_name=\"Basic managed network workspace example\",\n",
+    "    description=\"This example shows how to create a basic workspace with managed network enabled\",\n",
+    "    managed_network=managed_network,\n",
+    "    tags=dict(purpose=\"demo\"),\n",
+    ")\n",
+    "\n",
+    "ws_mvnet = ml_client.workspaces.begin_create(ws_mvnet).result()\n",
+    "print(ws_mvnet)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1.5 Update the isolation mode to AllowOnlyApprovedOutbound (AOAO) \n",
+    "The isolation mode for the workspace managed network can be updated using workspace update to be the more restrictive AOAO mode. After this update in mode, we can see the difference in outbound rules of the managed network. Some additional rules for system communication are added to the network settings as `Required` outbound rules. We can also see that these required rules are `Inactive` and the managed network is also in `Inactive` state as managed network creation is deferred.\n",
+    "\n",
+    "Using the `MLClient` created earlier, we will update the workspace. This command will start workspace update and provide a confirmation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name": "basic_ex_workspace_name"
+   },
+   "outputs": [],
+   "source": [
+    "ws_mvnet = ml_client.workspaces.get(mvnet_workspace_name)\n",
+    "\n",
+    "ws_mvnet.managed_network.isolation_mode = IsolationMode.ALLOW_ONLY_APPROVED_OUTBOUND\n",
+    "\n",
+    "aoao_ws = ml_client.workspaces.begin_update(ws_mvnet).result()\n",
+    "\n",
+    "print(aoao_ws)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2. Provisioning the Network\n",
+    "The managed network resources will be deferred on creation of the workspace, but we can force creation of them by calling the provision network command (a step that is required with spark enabled flag for spark training jobs - more on that in the docs).\n",
+    "\n",
+    "We can see that after provisioning the network, the status of the network and the outbound rules is `Active`.\n",
```

Comment:
Maybe specify the common reasons why it's not active? OR why active means for PE rules (pe created, in success state and approved)

Reply:
if rules inactive after provision, its a service bug and we should fix it. we should not include that comment in example documentations. 

included comment on what rules being active means:

"means that the rules are effective in the managed network (PEs created, NSG rules added for service tags, FQDN rules applied via managed firewall)."